src/compas/datastructures/mesh/core/attributes.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+try:
+    from collections.abc import MutableMapping  # python > 3.3
+except ImportError:
+    from collections import MutableMapping      # python 2.7
+
+
+__all__ = ['VertexAttributeView', 'FaceAttributeView', 'EdgeAttributeView']
+
+
+class AttributeView(object):
+    """Mixin for attribute dict views."""
+
+    def __init__(self, defaults, attr, key, custom_only):
```

Comment:
the default for custom-only should be false.

Reply:
Changed that. Thanks.